build/rules.mak

```diff
@@ -60,14 +60,14 @@ module: $(MODULE_FILES)
 
 $(UNIT_TEST_FILES): $(MODULE_FILES) $(RST_FILES)
 
-unit_tests: $(UNIT_TEST_FILES)
+unit_tests: $(UNIT_TESTS_PASSED)
 
-$(LOG_DIR)/tests_passed: $(UNIT_TEST_FILES)
+$(UNIT_TESTS_PASSED): $(UNIT_TEST_FILES)
 	$(call trace_to_console, "Running pytest",$@)
```

Comment:
I did realize that in the output of make, this prints:

` Running pytest: bin/nifake/log/tests_passed
`

It's a little bit awkward...

Reply:
We can change this to something better in a later change, after deciding what "better" would mean for this.